dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsApiServiceImpl.java

```diff
@@ -577,11 +605,23 @@ private Response getFileByToolVersionID(String registryId, String versionId, Sou
         if (first.isPresent() && oldFirst.isPresent()) {
             final ToolVersion toolVersion = first.get();
             final String toolVersionName = toolVersion.getName();
-            if (type == DOCKERFILE) {
+            switch (type) {
+            case WDL_TEST_JSON:
+            case CWL_TEST_JSON:
+                final EntryVersionHelper<Tool> entryVersionHelper = new EntryVersionHelper<>(toolDAO);
+                SourceFile sourceFile = entryVersionHelper.getSourceFile(entry.getId(), versionId, type);
```

Comment:
Can we get some junit tests on this? 
In particular, let's test what this looks like when hitting the GA4GH endpoints.
Also, does this support multiple test json for one tool version?